lib/miq_automation_engine/engine/miq_ae_state_machine.rb

```diff
@@ -74,7 +74,7 @@ def process_state(f, message, args)
 
         # Check the ae_result and set the next state appropriately
         if   @workspace.root['ae_result'] == 'ok'
-          $miq_ae_logger.warn "Processed State =[#{f['name']}]"
+          $miq_ae_logger.info "Processed State =[#{f['name']}]"
         elsif @workspace.root['ae_result'] == 'skip'
           $miq_ae_logger.warn "Skipping State =[#{f['name']}]"
```

Comment:
Should skip be info too? It sounds equally deliberate/expected/unconcerning to me.